src/CalcManager/Header Files/EngineStrings.h

```diff
@@ -214,7 +165,8 @@
 #define SIDS_ERR_INPUT_OVERFLOW L"119"
 #define SIDS_ERR_OUTPUT_OVERFLOW L"120"
 
-__declspec(selectany) std::wstring g_sids[] =
+// Include the resource key ID from above into this vector to load it into memory for the engine to use
+__declspec(selectany) std::vector<std::wstring> g_sids =
```

Comment:
Some of the function require these to be convertible to wstring or convertible to Platform::String, can this be done with wstring_view?

Reply:
I would update those functions to `wstring_view` as appropriate.  If you need a `wstring`, you can construct one from a `wstring_view`:
```C++
wstring foo(const wstring_view& sv) {
    return wstring{ sv };
}
```

Reply:
I made the change, I didn't update GetCEngineString to accept a wstring_view since it would need to call c_str() anyway to pass a Platform::String to the ResourceLoader